internal/restic/node_test.go

```diff
@@ -205,8 +206,18 @@ func TestNodeRestoreAt(t *testing.T) {
 			var nodePath string
 			if test.ExtendedAttributes != nil {
 				if runtime.GOOS == "windows" {
-					// restic does not support xattrs on windows
-					return
+					// In windows extended attributes are case insensitive and windows returns
+					// the extended attributes in UPPER case.
+					// Update the tests to use UPPER case xattr names for windows.
+					extAttrArr := test.ExtendedAttributes
+					// Iterate through the array using pointers
+					for i := 0; i < len(extAttrArr); i++ {
+						// Get the pointer to the current element
+						namePtr := &extAttrArr[i].Name
```

Comment:
I'd prefer the following (not really sure why, but it's better at hiding the pointer ^^ ):
```
entry := &extAttrArr[i]
entry.Name = strings.ToUpper(entry.Name)
```
or simply
`extAttrArr[i].Name = strings.ToUpper(extAttrArr[i].Name)`

Reply:
Updated thanks